### PR TITLE
deps: update to node RC for Arabica

### LINKED ISSUE #123: celestia node v0.11.0: breaking changes
TODO

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

This PR updates Celestia Node to Arabica and to v0.11.0-rc1. 

Resolves https://github.com/celestiaorg/optimism/issues/123

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by jcstein** [comment]
[v0.11-logs.log](https://github.com/celestiaorg/optimism/files/11606471/v0.11-logs.log)

I'm testing out this branch using a local light node for now running on https://github.com/celestiaorg/celestia-node/pulls/2281 on Arabica. It looks like we'll have to change the length of namespace for new schema:

```Shell
[32mop-batcher_1   |[0m t=2023-05-30T22:18:08+0000 lvl=warn msg="unable to publish tx to celestia" service=batcher err="unsupported namespace id length: id [232 229 246 121 191 113 22 203] must be 28 bytes but it was 8 bytes"
[32mop-batcher_1   |[0m t=2023-05-30T22:18:08+0000 lvl=warn msg="unable to publish tx"             err="unsupported namespace id length: id [232 229 246 121 191 113 22 203] must be 28 bytes but it was 8 bytes" data_size=6746
[32mop-batcher_1   |[0m t=2023-05-30T22:18:08+0000 lvl=warn msg="Failed to send transaction"       err="unsupported namespace id length: id [232 229 246 121 191 113 22 203] must be 28 bytes but it was 8 bytes"
```

cc @tuxcanfly

**Comment by tuxcanfly** [comment]
Using a versioned namespace, `V0` we just need 10 bytes so updated to use that.

**Review by tuxcanfly — APPROVED** [review]
OK.

File: ops-bedrock/docker-compose-devnet.yml [hunk 1]
```diff
@@ -83,8 +83,8 @@ services:
       --metrics.port=7300
       --pprof.enabled
       --rpc.enable-admin
-      --da-rpc=http://da:26659
-      --namespace-id=e8e5f679bf7116cb
+      --da-rpc=http://host.docker.internal:26659
```
Comment (tuxcanfly) [inline ops-bedrock/docker-compose-devnet.yml]: Is the change to `http://host.docker.internal` required?

**Review by tuxcanfly — CHANGES_REQUESTED** [review]

File: ops-bedrock/docker-compose-devnet.yml (hunk 1 above)
Comment (jcstein) [reply ops-bedrock/docker-compose-devnet.yml]: no this was just for testing yesterday, didn't mean to still have this there. fixing

File: ops-bedrock/docker-compose-testnet.yml [hunk 2]
```diff
@@ -122,8 +122,8 @@ services:
       OP_PROPOSER_L2OO_ADDRESS: "${L2OO_ADDRESS}"
       OP_PROPOSER_PPROF_ENABLED: "true"
       OP_PROPOSER_METRICS_ENABLED: "true"
-      OP_PROPOSER_ALLOW_NON_FINALIZED: "true"
-      OP_PROPOSER_NAMESPACE_ID: "e8e5f679bf7116cb"
+      OP_PROPOSER_ALLOW_NON_FINALIZED: "false"
```
Comment (tuxcanfly) [inline ops-bedrock/docker-compose-testnet.yml:125]: This seems unrelated.

**Review by tuxcanfly — CHANGES_REQUESTED** [review]

File: ops-bedrock/docker-compose-testnet.yml (hunk 2 above)
Comment (jcstein) [reply ops-bedrock/docker-compose-testnet.yml:125]: this was to test out the same changes as #133

Comment (jcstein) [reply ops-bedrock/docker-compose-testnet.yml:125]: and would revert before merging

**Comment by jcstein** [comment]
[v0.11.0-rc1-opstack.log](https://github.com/celestiaorg/optimism/files/11619572/v0.11.0-rc1-opstack.log)

The logs before bumping gas fee https://github.com/celestiaorg/optimism/pull/132/commits/7c2a2a2dfacae21c33b7413218c8e86b8e07a0de

File: ops-bedrock/docker-compose-testnet.yml (hunk 2 above)
Comment (jcstein) [reply ops-bedrock/docker-compose-testnet.yml:125]: https://github.com/celestiaorg/optimism/pull/132/commits/feaa7a9cfd22f8a5726d21fa633f3930524ac90c

**Review by tuxcanfly — APPROVED** [review]
OK.

**Comment by jcstein** [comment]
```bash
root@jcs-optimism:~/celestia-node# cast block safe --rpc-url http://localhost:9545 --field number
3836
root@jcs-optimism:~/celestia-node# cast block latest --rpc-url http://localhost:9545 --field number
3851
```

**Comment by jcstein** [comment]
# 4200+ safe blocks

At 4200+ safe blocks it seems that the rollup is stable on top of Arabica using a light node!

```bash
root@jcs-optimism:~/celestia-node# cast block latest --rpc-url http://localhost:9545 --field number
4269
root@jcs-optimism:~/celestia-node# cast block safe --rpc-url http://localhost:9545 --field number
4256
```

## Full logs

[v0.11-logs-6bbc6e0.log](https://github.com/celestiaorg/optimism/files/11620183/v0.11-logs-6bbc6e0.log)

## Interacting with smart contract

Works as expected:

```bash
# get count in hex
root@jcs-optimism:~/gm-portal/contracts# cast call $CONTRACT_ADDRESS "getTotalWaves()" --rpc-url $RPC
0x0000000000000000000000000000000000000000000000000000000000000005

# send new gm "wave" to contract 
root@jcs-optimism:~/gm-portal/contracts# cast send $CONTRACT_ADDRESS "wave(string)" "gm" --private-key $PRIVATE_KEY --rpc-url http://localhost:9545

blockHash               0x84e7fdcaa8d3d4573bf539165ba7eee4f897e734a9d804869eea1d9ce52735ee
blockNumber             4349
contractAddress
cumulativeGasUsed       154204
effectiveGasPrice       3000000008
gasUsed                 107363
logs                    [{"address":"0x22d4e211ef8704f2ca2d6dfdb32125e2530ace3e","topics":["0x5f7e16dc676677766a70e9c5628aa6c54ddb8b6e5188e2ae1e1f17f1ffbea716","0x000000000000000000000000de3829a23df1479438622a08a116e8eb3f620bb5"],"data":"0x000000000000000000000000000000000000000000000000000000006477fe3c00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000002676d000000000000000000000000000000000000000000000000000000000000","blockHash":"0x84e7fdcaa8d3d4573bf539165ba7eee4f897e734a9d804869eea1d9ce52735ee","blockNumber":"0x10fd","transactionHash":"0x6c36a01cd29bc2e2ed975b073770f55cb7aa44e8dd8bc8893be58e2ab163ca02","transactionIndex":"0x1","logIndex":"0x0","removed":false}]
logsBloom               0x00000000000000000000000000000000000000000000040000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000010000000000000080000040000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000
root
status                  1
transactionHash         0x6c36a01cd29bc2e2ed975b073770f55cb7aa44e8dd8bc8893be58e2ab163ca02
transactionIndex        1
type                    2

# check number of "waves" again
root@jcs-optimism:~/gm-portal/contracts# cast call $CONTRACT_ADDRESS "getTotalWaves()" --rpc-url $RPC
0x0000000000000000000000000000000000000000000000000000000000000006
```

## Previous errors

No previous errors occurred 🎉
